doc/source/whatsnew/v0.19.0.txt

```diff
@@ -731,3 +731,5 @@ Bug Fixes
 - Bug where ``pd.read_gbq()`` could throw ``ImportError: No module named discovery`` as a result of a naming conflict with another python package called apiclient  (:issue:`13454`)
 - Bug in ``Index.union`` returns an incorrect result with a named empty index (:issue:`13432`)
 - Bugs in ``Index.difference`` and ``DataFrame.join`` raise in Python3 when using mixed-integer indexes (:issue:`13432`, :issue:`12814`)
+
+- Bug in ``Series.sort_index`` didn't accept kwargs ``kind`` and ``na_position`` (:issue:`13589`)
```

Comment:
its not a bug, rather an API change, where ``Series.sort_index`` gained these kwargs (they are somewhat similar, but in this case we are highliting the change).